addons/hr_attendance/security/hr_attendance_security.xml

```diff
@@ -17,13 +17,13 @@
     </record>
 
     <record id="group_hr_attendance" model="res.groups">
-        <field name="name">Manual Attendance</field>
+        <field name="name">User : Only kiosk mode</field>
```

Comment:
Shouldn't the name of this group and the one above be inverted? :thinking:

Reply:
Indeed, it makes more sens that way :smile: